src/main/scala/uk/gov/hmrc/ct/computations/CP286a.scala

```diff
@@ -30,7 +30,9 @@ case class CP286a
 
   override def validate(boxRetriever: ComputationsBoxRetriever): Set[CtValidation] = {
     collectErrors(
-      requiredErrorIf(value.isEmpty && boxRetriever.cpQ18().isTrue),
+      requiredErrorIf(value.isEmpty && boxRetriever.cpQ18().isTrue
+        && mayHaveNirLosses(boxRetriever) &&
+        !hasValue),
```

Comment:
If this box is new with NIR then this condition is not right.

Reply:
This box is new with the new NI Journey.